solutions/usaco-743.mdx

```diff
@@ -0,0 +1,87 @@
+---
+id: usaco-743
+source: USACO Silver 2017 Open
+title: Modern Art 2
+author: Sofia Yang
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_art2_gold_open17.html)
+
+
+### Implementation
+
+**Time Complexity:** $\mathcal{O}(N)$
+
+<LanguageSection>
+<JavaSection>
+
+```java
+
```

Comment:
```suggestion
```

Reply:
fyi this doesn't make the file not render on github (extra line after the java opening tag)